portal/db.py

```diff
@@ -71,6 +72,29 @@ def mock_db():
             with con.cursor() as cur:
                 cur.execute(f.read())
 
+def import_csv():
+    """ import csv """
+    file = os.path.join(os.path.dirname(__file__), os.pardir, "portal_users.csv")
+    with open(file) as sql:
+        with get_db() as con:
+            with con.cursor() as cur:
+
+                cur.copy_from(sql, 'users', sep=",")
+
+                cur.execute("SELECT password FROM users")
+                passwords = cur.fetchall()
+                for password in passwords:
+                    cur.execute("UPDATE users SET password = %s WHERE password = %s", (generate_password_hash(password[0]), password[0]))
+
+                sql.close()
+
```

Comment:
Note: since you are opening this file using the context manager (`with open():`) it automatically closes at the end of the block.